index.bs

```diff
@@ -1150,17 +1152,21 @@ to render as though it was not stopped.
 
 # Referencing a SIMID creative from VAST # {#api-vast}
 
-When a SIMID creative is referenced from a VAST document, the value for the
-`apiFramework` attribute in the `InteractiveCreativeFile` element must be
-`SIMID` (all caps). This attribute identifies the SIMID API for the creative.
-Version negotiation should be handled by the [[#api-ad-loading]] algorithm
-(rather than identified in the VAST file).
-
-Another attribute of the `InteractiveCreativeFile` is `variableDuration`.
-If `variableDuration` is true this means that the ad is only playable if
-the player allows the ad to pause and extend duration. If the player does
-not support this capability, then it must not render the video or creative.
-The player should error out instead.
+When a SIMID creative is referenced within a VAST document, the `<InteractiveCreativeFile>` element must include
+the following required attributes on the element: `type="text/html"` and `apiFramework="SIMID"`.
+
+The value of the `apiFramework` attribute identifies SIMID as the required API for the creative. Players that do not support the SIMID API may load
+an audio or video file included with the ad, but they will not load or play the SIMID creative.
+
+Media players that do support the SIMID API should handle version negotiation between the creative and the media player via the
+[[#api-ad-loading]] algorithm. The SIMID API version is not identified by any element or attribute in the VAST file.
+
+A third, optional attribute which may be included on the InteractiveCreativeFile element is `variableDuration="true"`.
+If present, this attribute indicates that the ad unit is only playable if
+the media player allows the creative to pause playback of the ad's audio or video and extend the duration of the ad break (for example, with
+interactive content such as a game or survey). If the player does not support or allow this capability, then it **must not** render
+the current ad's audio/video or SIMID creative. The player should error out the ad instead (and either resume its primary content or continue
+on to the next ad in the current ad pod).
 
```

Comment:
The above section felt a bit thin to me as it was, and it also seemed like the proper place to move some of the technical details that were in the Intro section (originally with the <InteractiveMediaFile> example code). I hope you'll agree that this all works, but feel free to suggest revisions.

Reply:
I think this is good copy.

Reply:
Thanks, glad you approve! :-D